serf/serf.go

```diff
@@ -25,7 +25,7 @@ import (
 // version to memberlist below.
 const (
 	ProtocolVersionMin uint8 = 2
-	ProtocolVersionMax       = 4
+	ProtocolVersionMax       = 5
```

Comment:
One other thing to check is there anything to worry about with Serf itself. We might want to keep [this](https://github.com/hashicorp/serf/blob/master/serf/config.go#L243) set to 4. For consul I make a constant like `ProtocolVersion4Compatible = 4` to signal that we aren't taking the max.